src/components/BreadCrumbs/BreadCrumbs.tsx

```diff
@@ -0,0 +1,47 @@
+import { Link as RouterLink, useLocation } from 'react-router-dom';
+import Link from '@mui/material/Link';
+import Typography from '@mui/material/Typography';
+import Breadcrumbs from '@mui/material/Breadcrumbs';
+import { uuidExtract } from '../../utils/extract';
+import { ucFirstText } from '../../utils/text';
+import { NavigateNext } from '@mui/icons-material';
+import { StyledRoutedLinkProps } from '../../types/types';
+
+function StyledRoutedLink(props: StyledRoutedLinkProps) {
+  return <Link {...props} component={RouterLink as any} />;
+}
+
+export default function BreadCrumbs() {
+  const location = useLocation();
+  const pathnames = location.pathname.split('/').filter((x) => x);
```

Comment:
Why is the filter needed?

Reply:
To remove undefined pathnames.
For example
`/zaaktypen/:id/` will result in `['zaaktype', ':id', '']`